cdc/sink/producer/kafka/config.go

```diff
@@ -226,31 +226,47 @@ func newSaramaConfig(ctx context.Context, c *Config) (*sarama.Config, error) {
 		return nil, errors.Trace(err)
 	}
 	config.Version = version
-	// See: https://kafka.apache.org/documentation/#replication
-	// When one of the brokers in a Kafka cluster is down, the partition leaders
-	// in this broker is broken, Kafka will election a new partition leader and
-	// replication logs, this process will last from a few seconds to a few minutes.
-	// Kafka cluster will not provide a writing service in this process.
-	// Time out in one minute.
-	config.Metadata.Retry.Max = 120
-	config.Metadata.Retry.Backoff = 500 * time.Millisecond
+
+	// TODO: make these configurations can be customized by user
+	// Producer fetch metadata from brokers frequently, if metadata cannot be
+	// refreshed, this would indicate bad network connection between the capture
+	// server and kafka broker. The default setting should be enough to detect
+	// the health of kafka cluster. For kafka cluster with a bad network condition,
+	// errors can be quickly obtained.
+	// In the scenario that kafka cluster is down, this would cost at most around
+	// 750ms to get known that.
+	config.Metadata.Retry.Max = 3
+	config.Metadata.Retry.Backoff = 250 * time.Millisecond
 	// If it is not set, this means a metadata request against an unreachable
 	// cluster (all brokers are unreachable or unresponsive) can take up to
 	// `Net.[Dial|Read]Timeout * BrokerCount * (Metadata.Retry.Max + 1) +
-	// Metadata.Retry.Backoff * Metadata.Retry.Max`
-	// to fail.
+	// Metadata.Retry.Backoff * Metadata.Retry.Max` to fail.
 	// See: https://github.com/Shopify/sarama/issues/765
 	// and https://github.com/pingcap/tiflow/issues/3352.
+	// In the scenario that the network connection `producer -> kafka` is ok,
+	// but the producer cannot get response from kafka, this is the upperbound
+	// time cost for fetching metadata.
 	config.Metadata.Timeout = 1 * time.Minute
 
+	// Admin.Retry take effect on `ClusterAdmin` related operations,
+	// only `CreateTopic` for cdc now. Just use default values.
+	config.Admin.Retry.Max = 5
+	config.Admin.Retry.Backoff = 100 * time.Millisecond
```

Comment:
It looks like we removed its total timeout, what is the impact of the removed configuration on call to the admin API?

Reply:
it's still tunning.

This would works for only `CreateTopic` logic at the moment by `cdc cli`, does not affect runtime behavior.

Reply:
This only affect when try to `CreateTopic`, does not affect running behavior.